src/views/Home/components/CardBusdValue.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { useEffect, useRef } from 'react'
+import { useCountUp } from 'react-countup'
```

Comment:
there is already a component making use of `useCountUp`, a lot of your code is duplicated here